lib/node_modules/@stdlib/array/complex64/benchmark/benchmark.every.js

```diff
@@ -0,0 +1,55 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var isComplex64 = require( '@stdlib/assert/is-complex64' );
+var isBoolean = require( '@stdlib/assert/is-boolean' ).isPrimitive;
+var pkg = require( './../package.json' ).name;
+var Complex64Array = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg+':every', function benchmark( b ) {
+	var bool;
+	var arr;
+	var i;
+
+	function predicate( v ) {
+		return isComplex64( v );
+	}
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		arr = new Complex64Array( [ i, i, i+1, i+1 ] );
+		bool = arr.every( predicate );
+		if ( typeof bool !== 'boolean' ) {
+			b.fail( 'should return a boolean' );
+		}
+	}
+	b.toc();
+	if ( !isBoolean( bool ) ) {
+		b.fail( 'should return a boolean' );
+	}
+	b.pass( 'benchmark finished' );
+	b.end();
```

Comment:
```suggestion
	b.tic();
	for ( i = 0; i < b.iterations; i++ ) {
		arr = new Complex64Array( [ i, i, i+1, i+1 ] );
		bool = arr.every( predicate );
		if ( typeof bool !== 'boolean' ) {
			b.fail( 'should return a boolean' );
		}
	}
	b.toc();
	if ( !isBoolean( bool ) ) {
		b.fail( 'should return a boolean' );
	}
	b.pass( 'benchmark finished' );
	b.end();
	
	function predicate( v ) {
		return isComplex64( v );
	}
```

Only in examples do we generally declare a function before use. Otherwise, we move to the end.